src/components/Icon/DotsIcon/DotsIcon.tsx

```diff
@@ -4,16 +4,36 @@ import Icon, { IIconProps, propTypes as iconPropTypes } from '../Icon';
 import { lucidClassNames } from '../../../util/style-helpers';
 import { omitProps } from '../../../util/component-types';
 import { Color } from '../Icon';
+import PropTypes from 'react-peek/prop-types';
 
 const cx = lucidClassNames.bind('&-DotsIcon');
 
-interface IDotsIconProps extends IIconProps {}
+const { oneOf } = PropTypes;
+
+enum Orientation {
+	vertical = 'vertical',
+	horizontal = 'horizontal'
+}
+interface IDotsIconProps extends IIconProps {
+	direction?: keyof typeof Orientation;
```

Comment:
instead of using an enum, i'd suggest just using string literals `'vertical' | 'horizontal'` based on how you are using it here.

if you expect consumers to import your enum type and pass a value for a prop, you could use `direction?: Orientation`, but I don't think that's what you want.

Reply:
Either way is fine with me. I've seen it done with both string literals and with enums in this codebase.

Reply:
`Legend` has an example of the string literal. I see that we use enums all over the place too.

Reply:
@jhsu @yasskin would you prefer then that we stick with string literals? I just changed it back, or rather consolidate both string literals into an "orientation enum" and use that in both places. I personally think using the enum is superior to using string literals, but this is your repo so it's your call.